day_7/10_little_monkeys.rb

```diff
@@ -0,0 +1,28 @@
+# get input
+print "How many monkeys? (under 10) > "
+number = gets.chomp.to_i
+puts ""
+
+# a has to map intergers to numbers in words
+numbers = {
+  1 => "One",
+  2 => "Two",
+  3 => "Three",
+  4 => "Four",
+  5 => "Five",
+  6 => "Six",
+  7 => "Seven",
+  8 => "Eight",
+  9 => "Nine",
+  10 => "Ten"
+}
+
+# an array in reverse for rhyme
+arr = (1..number).to_a.reverse
+
+# print each rhyme
+arr.each do |i|
```

Comment:
The nursery rhyme does change on the last line. How would you update your code to account for that?